flux-cli/src/main/java/com/marklogic/flux/impl/importdata/ImportFilesCommand.java

```diff
@@ -53,6 +55,17 @@ protected Dataset<Row> afterDatasetLoaded(Dataset<Row> dataset) {
         if (writeParams.extractText) {
             dataset = dataset.withColumn("extractedText", TextExtractor.build().apply(new Column("content")));
         }
+        Map<String, String> classifierOptions = writeParams.makeOptions();
+        if (classifierOptions.containsKey(Options.WRITE_CLASSIFIER_HOST) && !classifierOptions.get(Options.WRITE_CLASSIFIER_HOST).isEmpty()) {
```

Comment:
See my note below about testing. We can likely kill two birds with one stone here by having ClassifierParams have a "buildTextClassifier()" method. That would move all the logic here into ClassifierParams so that this class doesn't know how to build the UDF.

And it would give us a method we can unit-test to ensure that the UDF gets built correctly based on the CLI options - which is all we can really do here, since we can't depend on an API key to call S4. 

But as mentioned below - the text classifier UDF would really need to expose its config somehow so that a test could make sure it was properly configured.

Reply:
I made this possible and created a test. However, I don't particularly love what I had to make it possible. Please see the marklogic-spark-connector PR.
I'm happy to change this if there's a better way.

Reply:
I think I did this the way you want.